### PR TITLE
Feat/RTENU-34 landing page

### PR DESCRIPTION
**Pull request by NinaDang97**
**! For responsive testing on browser, remember to reload the page when move from mobile/tablet screen to desktop screen, and vice versa. Or open 2 separate tabs for mobile/tablet and desktop**

To-do:
- Add cypress testing
- Currently jest testing is not possible due to eslint parsing problem
- Proper authentication logic
- Proper check for `isFirstLogin`

### CODE REVIEW

File: packages/frontend/src/pages/Landing/index.tsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+export const Landing = () => {
+  // temporary state -> should save in db instead
+  const [isFirstLogin, setIsFirstLogin] = useState(() => {
+    const saved = localStorage.getItem('isFirstLogin') || 'true';
+    const initialValue = JSON.parse(saved);
+    return initialValue === 'true' || initialValue;
+  });
+
+  const location = useLocation();
+  console.log(location);
```
Comment (mehtis) [inline packages/frontend/src/pages/Landing/index.tsx:19]: console.log

File: packages/frontend/src/pages/Home/index.tsx [hunk 2]
```diff
@@ -1,22 +1,16 @@
...
+import { Routes } from '../../constants/Routes';
 
 export const Home = () => {
-  const location = useLocation();
   return (
     <div>
       <header>
         <img src={logo} alt="logo" />
-        <p>This is unstyled homepage (authentication will either redirect to here or to access-denied page)</p>
+        <p>Sinua ei ole todennettu!</p>
```
Comment (mehtis) [inline packages/frontend/src/pages/Home/index.tsx:10]: This sounds a bit off, but I can't really say what would be a better option 😄 Not a dealbreaker, we can modify it later.

**Review by mehtis — DISMISSED** [review]
There's that console log which should be removed but otherwise lgtm!

File: packages/frontend/src/pages/ProtectedPage/index.tsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+import { NavBar } from '../../components/NavBar';
+
+type Props = {
+  children: React.ReactElement;
+};
+
+export const ProtectedPage = ({ children }: Props) => {
+  // To-do: Authentication check goes here
+  // if unauthenticated --> logout
+  // if authenticated --> go to first login / landing page
```
Comment (saneDG) [inline packages/frontend/src/pages/ProtectedPage/index.tsx:14]:
```suggestion
```

**Review by saneDG — APPROVED** [review]

**Pull request merged**